vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -190,6 +191,9 @@ struct BridgeUIBatchInstanceCallback final : public facebook::react::InstanceCal
 // ReactInstanceWin implementation
 //=============================================================================================
 
+/*static*/ std::mutex ReactInstanceWin::s_mutex;
```

Comment:
>s_mutex [](http://example.com/codeflow?start=40&length=7)

(nit) is this supposed to protect only s_entries? If so, can you name more specifically, eg s_entriesMutex